Civi/Api4/Event/Subscriber/CustomValueAccessSubscriber.php

```diff
@@ -71,7 +71,11 @@ class_uses($apiRequest)
       return;
     }
 
-    $group = \CRM_Core_BAO_CustomGroup::getGroup(['name' => $apiRequest->getCustomGroup()]);
+    $group = \Civi\Api4\CustomGroup::get(FALSE)
+      ->addSelect('name', 'min_multiple', 'max_multiple')
+      ->addWhere('name', '=', $apiRequest->getCustomGroup())
+      ->addWhere('is_multiple', '=', TRUE)
```

Comment:
previously we didn't explicitly filter by `is_multiple` - though that was implied because we are in the context of a CustomGroup entity api call, the $apiRequest->getCustomGroup() will be a name for an `is_multiple` CustomGroup